### ISSUE TITLE
Question - PR

### ISSUE THREAD

**Issue by patchkez** [rack-roles/tower#6]
Hi Linuturk,
will you review/accept pull request against this repo if I add support for deployment of latest Ansible Tower on RHEL7/Centos7?

**Comment by Linuturk** [comment]
Absolutely! Pull requests are always welcome as long as they are tested and working :+1:
